project1.css

```diff
@@ -1,90 +1,395 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&display=swap');
+
 * {
-  font-family: 'Inter', sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+  font-family: "Inter", sans-serif;
 }
 
 header {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
-  align-items: center;
+  margin: 20px;
+}
+
+.welcome {
+  text-decoration: none;
+  color: black;
+  font-size: 1.25rem;
+  font-weight: 600;
+}
+
+button {
+  border: none;
+  color: #fff;
+  background: none;
+  font-weight: 500;
+  font-size: 17px;
+  text-align: center;
+  pointer-events: none;
+}
+
+.name {
+  margin-top: 10px;
+  color: #172b46;
 }
 
-@media (min-width: 350px) {
+.title {
+  text-align: center;
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 3.5rem;
+  width: 327px;
+}
+
+@media (max-width: 768px) {
```

Comment:
- [ ] We are not working yet with `@media`, but if you do it please 🙏🏽  make sure that looks good. kindly remove it until we require it.
![image](https://user-images.githubusercontent.com/58642949/192074651-ff81fce6-27c6-4b32-a248-27be54df5527.png)